src/main/scala/io/iohk/ethereum/network/package.scala

```diff
@@ -0,0 +1,20 @@
+package io.iohk.ethereum
+
+import io.iohk.ethereum.crypto._
+import org.spongycastle.crypto.params.{AsymmetricKeyParameter, ECPublicKeyParameters}
+import org.spongycastle.util.encoders.Hex
+
+package object network {
+
+  def nodeIdFromPublicKey(pubKey: AsymmetricKeyParameter) = {
```

Comment:
What about an implicit class extends AnyVal { def toNodeId: String } ?
Same for nodeId to public key?